library/src/main/res/drawable/collection_item_image_border.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+	<stroke android:width="0.5dp" android:color="@color/badgeImageBorder" />
```

Comment:
Do not use smaller than 1dp values, cause either way it will turn to 1 dp under the hood.

Devices convert dp values into pixels using (basically) the following formula:
px = (int)(scale * dp + 0.5)

So 0.5dp would result in {1px, 1px, 1px, 2px} for the above densities, as 1dp would be {1px, 2px, 2px, 3px}. A tiny value like 0.1dp would resolve to {1px, 1px, 1px, 1px} because anything less than 1 in the above formula resolves to a single pixel unless the value was explicitly 0dp (it's not possible to draw something thinner than a single pixel width).

Reply:
Ahh true. Most phones are xxxhdpi. So 1dp -> 3px, and 0.5dp -> 2px, so still a difference :)